.github/workflows/bundle-artifacts.yml

```diff
@@ -0,0 +1,83 @@
+name: bundle-artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'Optionally override which branch to build'
+        required: false
+        default: main
+      repository:
+        description: 'Optionally override from where to fetch the specified ref'
+        required: false
+        default: git-for-windows/git
+
+env:
+  GPG_OPTIONS: "--batch --yes --no-tty --list-options no-show-photos --verify-options no-show-photos --pinentry-mode loopback"
+  HOME: "${{github.workspace}}\\home"
+  USERPROFILE: "${{github.workspace}}\\home"
+  REPOSITORY: "${{github.event.inputs.repository}}"
+  REF: "${{github.event.inputs.ref}}"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  bundle-artifacts:
+    runs-on: windows-latest
```

Comment:
I am _so_ itching to move this to `ubuntu-latest`, to avoid paying the startup/runtime cost of the slower Windows agents... Maybe it works with just this change?

Reply:
That actually makes a ton of sense indeed. I'm not seeing anything in these scripts that `ubuntu-latest` wouldn't be able to do. We'll need to get rid of the `setup-git-for-windows-sdk` step and some other specific logic, but nothing major there. Let me try. Thanks for the suggestion!

Reply:
Ok, it's [almost working](https://github.com/git-for-windows/git-for-windows-automation/actions/runs/4107209885/jobs/7086405513). Getting this error:

```
error: gpg failed to sign the data
error: unable to sign the tag
```

Not a very helpful error message. Have you run into this before?

Reply:
This should help: https://github.com/git-for-windows/build-extra/pull/476

Reply:
That [helped a lot](https://github.com/git-for-windows/git-for-windows-automation/actions/runs/4113180504), but we're not there yet. The script tries to `cd` into `/usr/src/build-extra`, but that doesn't work as I cloned it into `$RUNNER_TEMP/build-extra` instead. When I let CI clone into `/usr/src/build-extra` in one of my first attempts yesterday, I got a [Permission Denied](https://github.com/git-for-windows/git-for-windows-automation/actions/runs/4107164239/jobs/7086298667#step:3:19) error. I still think `$RUNNER_TEMP/build-extra` would be a good place for this as it's guaranteed that we have write permissions in that folder. We'd need to update `please.sh` a bit though. WDYT?

Reply:
How about using https://github.com/git-for-windows/build-extra/blob/main/add-release-note.js instead of `please.sh mention`? It's the long-term plan to retire the latter in favor of the former, anyway.

Reply:
Neat!! Looks like [that perfectly](https://github.com/git-for-windows/git-for-windows-automation/actions/runs/4115147857/jobs/7103465335). Took a few attempts to get right, sorry for all the pushes in this PR 😅